website/docusaurus.config.js

```diff
@@ -41,6 +41,18 @@ module.exports = {
             to: '/docs/intro-weave-gitops/',
             from: ['/docs/getting-started'],
           },
+          {
+            to: '/docs/intro-weave-gitops/',
+            from: '/docs/0.6.2/getting-started'
```

Comment:
wondering if we can use a wildcard `*` here so it'll take care of all the past versions 😅